MEMBER_TRAVEL_FUND.md

```diff
@@ -106,7 +106,7 @@ Richard Littauer | Collab Summit Spring 2018 | Attendance & Collaborate | Berlin
 Dan Shaw | Spring 2018 Collaborator's Summit and Code & Learn | Attendance & WG facilitator | Berlin, Germany | May 30 - Jun 3 | $1500
 Benjamin Zaslavsky | Collab Summit Spring 2018 | Attendance - Inclusivity Initiative | Berlin, Germany | May 31 - Jun 1 2018 | $385.59
 Agiri Abraham JNR | Collab Summit Spring 2018 | Attendance - Inclusivity Initiative & Mentorship | Berlin, Germany | May 31 - Jun 3 2018 | $1765.00
-Ujjwal Sharma | Collab Summit Spring 2018 | Attendance & Collaborate | Berlin, Germany | May 31-June 1 2018 | $1500
+Ujjwal Sharma | Collab Summit Spring 2018 | Attendance & Collaborate | Berlin, Germany | May 31-June 1 2018 | $1490
```

Comment:
This does not match the $1384 in the updated description

Reply:
Oops, updating.